src/integrations/prefect-dask/prefect_dask/task_runners.py

```diff
@@ -149,6 +149,16 @@ def result(
             _result = run_coro_as_sync(_result)
         return _result
 
+    def add_done_callback(self, fn: Callable[[PrefectFuture], None]):
```

Comment:
I don't think it's necessary to add these methods for each task runner given that `PrefectWrappedFuture` already has it implemented